app/services/intercom_service.rb

```diff
@@ -0,0 +1,113 @@
+require 'intercom'
+
+class IntercomService
+  def self.create_intercom_message_from_email(incoming_email, fw_from_hub: false)
+    email = incoming_email.client&.intake&.email_address || incoming_email.sender
```

Comment:
since we have "incoming_email" which is an email object, I'd suggest changing variable "email" to email_address to make it more clear the distinction between the two variables.

Also, why attempt to look at incoming email.client.intake.email instead of just grabbing from sender? Will those emails ever be different? And if they are different, do we care?

Reply:
True! I think we should just grab it from the sender, not sure why we did it this way in the first place